app/assets/javascripts/initializers/runtime.js

```diff
@@ -39,9 +39,12 @@ class Runtime {
    * @returns {boolean} true if current environment support native features
    */
   static isNativeAndroid(namespace = null) {
-    const nativeCheck =
-      /DEV-Native-android|ForemWebView/i.test(navigator.userAgent) &&
-      AndroidBridge != undefined;
+    let nativeCheck = /DEV-Native-android|ForemWebView/i.test(navigator.userAgent);
+    try {
+      nativeCheck = nativeCheck && (AndroidBridge != undefined);
+    } catch (e) {
```

Comment:
Is this for catching the `ReferenceError` when `AndroidBridge` is not defined? Would the following also work without relying on exceptions?

```js
nativeCheck &&= typeof(AndroidBridge) === "function"
```

Reply:
Yeah, funny enough I think is an error that surfaced in recent updates of WebKit where checking for the variable raises an exception (I could swear this didn't happen before).

I'll definitely try this out and replace it if it's works well on all platforms. Thanks!  🙌🏼

Reply:
I took this and sent a separate PR (#11969) so the fix can be merged earlier rather than waiting for this PR to be merged. 

Thanks for the suggestion @citizen428!